pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -74,6 +60,15 @@ var (
 			Name: util.EnvoyRawBufferSocketName,
 		},
 	}
+
+	passThroughUpstreamBindConfig = &core.BindConfig{
```

Comment:
without https://github.com/istio/istio/pull/29528 this is not safe, anyone patching this field would modify global state

Reply:
Ah.. got it. Will revert this change.